### PR TITLE
Add analytics script to the app

### PR DESCRIPTION
**Pull request by ibarrae**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,8 +91,8 @@ Then, open `.env.local` and add the missing environment variables:
   sure to verify the email in the AWS SES dashboard.
 - `NEXT_PUBLIC_FACEBOOK_ID` - Facebook APP ID. You can get it
   [here](https://developers.facebook.com/docs/development/create-an-app).
-- `AUTH0_AUDIENCE` - The url for Auth0 audience. Normally has the following structure `https://your-tenant.auth0.com/api/v2/`
-
+- `AUTH0_AUDIENCE` - The url for Auth0 audience. Normally has the following
+  structure `https://your-tenant.auth0.com/api/v2/`
```
Comment (JoseRFelix) [inline README.md:95]: Seeing some formatting changes. Are we using the same prettier config?

File: src/components/common/Layout/Layout.tsx [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+  <>
+    <NextHead>
+      {constants.site.cloudflareAnalyticsId && (
+        <script
+          defer
+          src="https://static.cloudflareinsights.com/beacon.min.js"
+          data-cf-beacon={`{"token": "${constants.site.cloudflareAnalyticsId}"}`}
+        ></script>
+      )}
+    </NextHead>
```
Comment (JoseRFelix) [inline src/components/common/Layout/Layout.tsx:21]: I think it's better to have this in the `_app.tsx`.

**Review by JoseRFelix — CHANGES_REQUESTED** [review]
Looks good, thanks! Check out the comments I left.

File: README.md (hunk 1 above)
Comment (ibarrae) [reply README.md:95]: We should be but I might same some conflicts :disappointed: Thanks for letting me know of this :+1:

File: src/components/common/Layout/Layout.tsx (hunk 2 above)
Comment (ibarrae) [reply src/components/common/Layout/Layout.tsx:21]: :thinking: Thanks for letting me know of this, we actually have a `Head` component already that's being used in `_app.tsx` and `_document.tsx`, would it be better to put it under there?

File: README.md (hunk 1 above)
Comment (JoseRFelix) [reply README.md:95]: :+1:

File: src/components/common/Layout/Layout.tsx (hunk 2 above)
Comment (JoseRFelix) [reply src/components/common/Layout/Layout.tsx:21]: Yes! That works, thanks

File: README.md [hunk 3]
```diff
@@ -75,6 +75,7 @@ Then, open `.env.local` and add the missing environment variables:
 - `NEXT_PUBLIC_API_URL` - The url of Next.js serverless functions
 - `NEXT_PUBLIC_API_WEBSOCKET_URL` - The url for websocket API in AWS.
 - `NEXT_PUBLIC_CLOUDFRONT_URL` - AWS Cloudfront url
+- `NEXT_PUBLIC_CLOUDFLARE_ANALYTICS_ID`: The token for Cloudflare analytics that can be retrieved from the [dashboard](https://dash.cloudflare.com).
```
Comment (JoseRFelix) [inline README.md:78]: Not a big deal, but we should use a `-`

**Review by JoseRFelix — APPROVED** [review]
LGTM! :rocket: (I left a little nitpick though)

**Pull request merged**